packages/flutter/lib/src/cupertino/magnifier.dart

```diff
@@ -0,0 +1,305 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'package:flutter/widgets.dart';
+
+/// A [CupertinoMagnifier], specifically for text editing.
```

Comment:
Nit: Should this line reflect that `SelectionArea` and `SelectableText` are supported (non text-editing widgets)?